src/ui/public/timepicker/timepicker.js

```diff
@@ -35,10 +37,10 @@ module.directive('kbnTimepicker', function (timeUnits, refreshIntervals) {
     template: html,
     controller: function ($scope) {
       $scope.format = 'MMMM Do YYYY, HH:mm:ss.SSS';
-      $scope.modes = ['quick', 'relative', 'absolute'];
+      $scope.modes = [TIME_MODES.QUICK, TIME_MODES.RECENT, TIME_MODES.RELATIVE, TIME_MODES.ABSOLUTE];
```

Comment:
I'd prefer the following:

```javascript
$scope.modes = Object.values(TIME_MODES);
```

Then, if someone adds a new mode for some reason, this line doesn't have to change, and it's more concise.

Reply:
We're using a similar pattern in EUI.